cdci_data_analysis/analysis/parameters.py

```diff
@@ -382,30 +393,34 @@ def value(self):
 
     @value.setter
     def value(self, v):
-        if v is not None and v!='':
+        if v is not None and v != '':
             self.check_float_value(v, name=self.name, units=self.units)
             self._v = np.float(v)
 
         else:
             self._v=None
 
+    def get_value_in_default_format(self, value):
+        return float(value)
+
     @staticmethod
     def check_float_value(value, units=None, name=None):
         #print('check type of ',name,'value', value, 'type',type(value))
-        if value is None or value=='':
+        if value is None or value == '':
             pass
         else:
             try:
-              value=ast.literal_eval(value)
+                value = ast.literal_eval(value)
```

Comment:
I do not understand at all what is the purpose of this. It's a very peculiar way to convert time type, effectively discarded by the rest of the method.

Reply:
I think it was there because, in case values in a string format (eg '1234') would be correctly converted ( 1234).

That's the explanation I give

Reply:
in any case, both here as well as for the `Integer` class, it can be removed I think